Makefile

```
@@ -649,6 +649,14 @@ bazel_bin: ## Build importer/tidb binary files with Bazel build system
  	cp -f ${TIDB_SERVER_PATH} ./bin/ ; \
  	cp -f ${IMPORTER_PATH} ./bin/ ;
 
+.PHONY: bazel_bin
+bazel_bin: ## Build importer/tidb binary files with Bazel build system
```

checkmake (0.3.2)

[warning] Target "bazel_bin" defined multiple times (lines 645 and 653).

(uniquetargets)